### ISSUE TITLE
questions about the paper

### ISSUE THREAD

**Issue by yutseng318** [Holmes-Alan/Hierarchical-Back-Projection-Network-for-image-SR#2]
Hi, 
Just want to make sure if there's typo in the paper.
Figure 3. describe the structure of UBP and DBP.
Beside the Conv op in UBP states 2Mx2NxC, which is quit confusing for me.
As far as I have understood, this Conv downsample the Deconv(x) and would be subtracted with x to construct residual map. 
Shouldn't the shape be MxNx2C to fit the shape of x?
The same situation happens in DBP's Deconv.

**Comment by Holmes-Alan** [comment]
yes, we have modified this typo, and you can check the newest version for your research. Thank you
